modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/200_logs_datastream_defaults.yml

```diff
@@ -0,0 +1,111 @@
+---
+Verify default logs-*-* settings and mappings:
+  - do:
+      indices.create_data_stream:
+        name: logs-test-1
```

Comment:
nit: use some more realistic data stream names, for example: `logs-generic-default`